google/ggrc-core
# GGRC-7705/GGRC-7706/GGRC-7707 Create framework for Bulk complete modal

## Should be merged to "asmt-bulk" branch.



# Steps to test the changes

Precondition: you should have 1 <= assessments which should pass next rules:
- you must have "update" rights on these assessments (just create assessments and you'll immediately get "update" rights on the created assessment).
- assessments should not be Archived, 
- possible assessments statuses are 'In Progress' / 'Rework Needed' / 'Not Started'.

Open My Assessments or Audit -> Assessments tab and click Bulk Complete button (on Audit page, it's a menu item of 3 dots menu).

# Solution description

What was done and should be tested.

1) By clicking on "Bulk complete" buttons, modal "Bulk complete" should be shown. 
2) Modal should have 3 sections as shown on the screenshot:

![image](https://user-images.githubusercontent.com/13063442/64346975-cdeb4680-cffb-11e9-98ea-d8558c45bf28.png)

3) "Select assessments" should be expanded by default after opening of the modal and default filter should be appllied - on Audit page, search should be relevant to opened Audit, on My Assessments page - relevant to the current logged user.

4) Advanced search was integrated to the modal. Need to test it. 

5) By default, statuses are in 'In Progress' / 'Rework Needed' / 'Not Started'.


# Sanity checklist




# PR Review checklist
